### PR TITLE
[16.0][MIG] partner_contact_gender

### PR DESCRIPTION
**Pull request by carmenbianca**
Ping @PierrickBrun

### CODE REVIEW

**Comment by marielejeune** [comment]
Depends on https://github.com/OCA/partner-contact/pull/1377

**Review by marielejeune — APPROVED** [review]
Code review done

**Review by PierrickBrun — APPROVED** [review]
code review + test ( with https://github.com/OCA/partner-contact/pull/1377)

**Review by AnizR — APPROVED** [review]

**Comment by carmenbianca** [comment]
Sadness :( I'll do it manually then.

**Comment by NL66278** [comment]
@carmenbianca You can rebase it locally and then force push

**Comment by carmenbianca** [comment]
@gurneyalex Same CLA issue as #1368

**Comment by gurneyalex** [comment]
CLA checked manually

File: partner_contact_gender/i18n/partner_gender.pot [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Translation of Odoo Server.
```
Comment (gurneyalex) [inline partner_contact_gender/i18n/partner_gender.pot]: can you remove this file?

**Review by gurneyalex — APPROVED** [review]

File: partner_contact_gender/i18n/partner_gender.pot (hunk 1 above)
Comment (carmenbianca) [reply partner_contact_gender/i18n/partner_gender.pot]: Done, thank you!

**Pull request merged**